showcase/app/templates/components/alert.hbs

```diff
@@ -244,4 +244,27 @@
     </SG.Item>
   </Shw::Grid>
 
+  <Shw::Text::H2>With a custom title tag</Shw::Text::H2>
```

Comment:
thought: it probably makes more sense for this section to live under 'Content', with a `Shw::Text::H3`